dotnet/src/Planners/Planners.Core.UnitTests/Handlebars/HandlebarsTemplateEngineExtensionsTests.cs

```diff
@@ -213,41 +214,41 @@ public void ShouldThrowExceptionWhenMissingRequiredParameter()
     {
         // Arrange
         var kernel = this.InitializeKernel();
-        var executionContext = kernel.CreateNewContext();
+        var contextVariables = new ContextVariables();
         var template = "{{Foo-Combine x=\"Bar\"}}";
         var variables = new Dictionary<string, object?>();
         kernel.ImportPluginFromObject(new Foo(), "Foo");
 
         // Assert   
-        Assert.Throws<SKException>(() => HandlebarsTemplateEngineExtensions.Render(kernel, executionContext, template, variables));
+        Assert.Throws<SKException>(() => HandlebarsTemplateEngineExtensions.Render(kernel, contextVariables, template, variables));
     }
 
     [Fact]
     public void ShouldThrowExceptionWhenFunctionHelperHasInvalidParameterType()
     {
         // Arrange
         var kernel = this.InitializeKernel();
-        var executionContext = kernel.CreateNewContext();
+        var contextVariables = new ContextVariables();
         var template = "{{Foo-StringifyInt x=\"twelve\"}}";
         var variables = new Dictionary<string, object?>();
         kernel.ImportPluginFromObject(new Foo(), "Foo");
 
         // Assert
-        Assert.Throws<ArgumentOutOfRangeException>(() => HandlebarsTemplateEngineExtensions.Render(kernel, executionContext, template, variables));
+        Assert.Throws<ArgumentOutOfRangeException>(() => HandlebarsTemplateEngineExtensions.Render(kernel, contextVariables, template, variables));
     }
 
     [Fact]
     public void ShouldThrowExceptionWhenFunctionHelperIsNotDefined()
     {
         // Arrange
         var kernel = this.InitializeKernel();
-        var executionContext = kernel.CreateNewContext();
+        var contextVariables = new ContextVariables();
         var template = "{{Foo-Random x=\"random\"}}";
         var variables = new Dictionary<string, object?>();
```

Comment:
`variables` here sounds confusing now that we have two variables with the same name.
maybe `templateVariables` ?

Reply:
The variables of context variables type will be either renamed or removed in one of the following PRs.